### ISSUE TITLE
🐛 Bug: Ersilia fetch should exit when fetching from DockerHub if Docker Engine is not running

### ISSUE THREAD

**Issue by DhanshreeA** [ersilia-os/ersilia#1404]
### Describe the bug.

If a model is being fetched from DockerHub especially with the `--from_dockerhub` flag, ersilia crashes if the engine is not running. Instead we should flag this and exit ersilia.

### Operating environment

Ubuntu 22.04 LTS (GitHub Actions)

**Comment by DhanshreeA** [comment]
This is also related to #1408

**Comment by DhanshreeA** [comment]
This fix is now implemented in #1445 and the behavior is depicted in the logs shared below. 

```bash
(ersilia) $ ersilia fetch eos80ch --from_dockerhub
⬇️  Fetching model eos80ch: malaria-mam
👎 Model eos80ch failed to fetch! Docker is not installed or active on your system.
```
